### DISCUSSION TITLE
How to get Profiler from console command ?

### DISCUSSION THREAD

**Discussion by HongKilDong** [symfony/symfony#57353] [Q&A] [answered]
Symfony 6.4 allow to profile console command with `--profile` option. I need to access profile data programmatically from console command and symfony declares this possibility for http requests: https://symfony.com/doc/6.4/profiler.html#accessing-profiling-data-programmatically. Profiler service marked as public, so I tried to get it as constructor argument of console command, but got error `... argument "$profiler" of method "__construct()" references class "Symfony\Component\HttpKernel\Profiler\Profiler" but no such service exists. You should maybe alias this class to the existing "profiler" service. `
So how to get profiler data programmatically from console command ?

**Comment by chapterjason** [comment, marked as answer]
Have you tried to add the attribute `#[Autowire('@profiler')]` to the param?
